aprsd_webchat_extension/web/chat/static/css/chat.css

```
@@ -284,3 +284,91 @@ input[type=search]::-webkit-search-cancel-button {
+.btn-raw-toggle.active {
+    color: var(--primary-color);
+}
+
+.btn-raw-toggle.active .material-symbols-rounded {
+    font-variation-settings: 'FILL' 1;
+}
+
+/* Raw packet text element (hidden by default) */
+.bubble-raw-packet {
+    display: none;
+    font-family: 'SF Mono', 'Monaco', 'Inconsolata', 'Roboto Mono', 'Courier New', monospace;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Monaco" (font-family-name-quotes)

(font-family-name-quotes)